antora-ui-camel/src/css/vars.css

```diff
@@ -54,6 +54,8 @@
   /* nav */
   --nav-background: var(--panel-background);
   --nav-border-color: var(--color-smoke-50);
+  --nav-scrollbar-color: var(--color-gray-10);
+  --nav-scrollbar-active-color: var(--color-gray-50);
```

Comment:
Can we make this on same lines and with the same name as in the [Default Antora UI](https://gitlab.com/antora/antora-ui-default/-/blob/master/src/css/vars.css), this would make it easier to rebase.

Reply:
Yes, I used it similar to it and included additional ones as well one for track and one for when the thumb of the scrollbar is active.